drupal/sites/all/features/stats_charts/stats_charts.panels_default.inc

```diff
@@ -38,7 +38,7 @@ function stats_charts_default_panels_mini() {
     $pane->access = array();
     $pane->configuration = array(
       'admin_title' => 'Project stats overview',
-      'title' => 'Einstein@Home Progress',
+      'title' => bts('Einstein@Home Progress', array(), NULL, 'boinc:frontpage'),
```

Comment:
"this" means the string that explicitly mentions the project. When another project is using the Drupal code this is the first thing they need to change. I don't know if it is possible to make the project name configurable in this file so that the string would be "@project-name Progress" and then it is ok to be part of the "boinc" textgroup.